docs/document_store_schema.md

```diff
@@ -0,0 +1,384 @@
+# Document Store Schema
+
+## Event
+An event can be a normally scheduled meeting, a special event such as a press conference or election debate, and, can be
+upcoming or historical.
+
+### Schema
+```
+event_id: {
+    body: {
+        id: str
+        name: str
+    }
+    event_datetime: datetime
+    thumbnail_static_file: {
+        id: str
+        uri: str
+    }
+    thumbnail_hover_file: {
+        id: str
+        uri: str
+    }
+    video_uri: optional[str]
+    keywords: [
+        {
+            id: str
+            phrase: str
+        }
+    ]
+    matters: [
+        {
+            id: str
+            name: str
+            decision: str
+        }
+    ]
```

Comment:
Could we add a `type` field? We need it to create the ACTION text in the event card: https://projects.invisionapp.com/share/UGW7LLV87VQ#/screens/407317059

Reply:
I added `type` and `title` fields to all collections that reference `matter` as just a standard.